docs/proposal-spawn-on-d.md

```diff
@@ -205,22 +205,51 @@ trigger events too, but for now a waiting task with an xtrigger is fine).
 
 ### Absolute dependence
 
-Here `foo.1,2,3,...` should trigger off of `start.2`:
+At first glance this is not a great fit for the spawn-on-demand model: 
 ```
 ...
       R1/2 = "start"
-      P1 = "start[2] => foo"
+      P1 = "start[2] => bar"
 ```
+Above, with no final cycle point a single event (`start.2 succeeded`) should
+spawn an infinite number of children (`bar.1,2,3,...`), which obviously is not
+feasible.
+
 (Note without the first line above `start` is not defined on any sequence, so
 with SoS the scheduler will stall with `foo` unsatisfied, but with SoD it will
 shut down immediately with nothing to do because and `foo` never gets spawned.)
```

Comment:
```suggestion
with SoS the scheduler will stall with `bar` unsatisfied, but with SoD it will
shut down immediately with nothing to do because and `bar` never gets spawned.)
```

Reply:
Good spotting @wxtim . I had changed `foo` to `bar` in the example, to be more like the new example added below it :grimacing: